scripts/provision-namespace.sh

```diff
@@ -0,0 +1,184 @@
+#!/bin/bash
+
+execute_kubectl() {
+  local command=$1
+  if $DRY_RUN; then
+    echo "kubectl --dry-run=client $command "
+    kubectl --dry-run=client $command
+  else
+    echo "kubectl $command"
+    kubectl $command
+  fi
+}
+
+execute_oc() {
+  local command=$1
+  if $DRY_RUN; then
+    echo "oc $command "
+  else
+    echo "oc $command"
+    oc $command
+  fi
+}
+
+# Set default values
+DRY_RUN=false
+
+# Display usage
+function usage() {
+  echo "Usage: $0 [options]"
+  echo "Options:"
+  echo "  -n, --namespace     <namespace>                        The namespace on the QShift cluster (mandatory)"
+  echo "  -q, --quay          <quay_username:quay_password>      The Quay username and password (mandatory)"
+  echo "  -o, --quay-org      <quay_username:quay_password>      The Quay organization hosting the images (mandatory)"
+  echo "  -d, --docker        <docker_username:docker_password>  The docker username and password (mandatory)"
```

Comment:
I shouldn't be needing both quay and docker. One of the two should do.

Reply:
Also, both docker and quay can be read from `$HOME/.docker/config.json` we should try that if no parameter has been specified.

Reply:
Good idea to also pass as parameter the config.json file.
 
Remark: Such a file to be imported will not work for macos users as docker manage the creds using OSKEYSTORE
```
cat ~/.docker/config.json
{
	"auths": {
		"ghcr.io": {},
		"https://index.docker.io/v1/": {},
		"quay.io": {}
	},
	"credsStore": "osxkeychain",
```